vlib/v/gen/c/testdata/comptime_if_nix.vv

```diff
@@ -0,0 +1,9 @@
+platform := $if windows {
```

Comment:
```suggestion
platform := $if some_custom_define ? {
```

Reply:
(and it will force the generation of the bottom else case, where it failed before)